installer/tue-install-impl.bash

```diff
@@ -810,15 +810,18 @@ function _tue-install-pip
     local pv=$1
     shift
     tue-install-debug "tue-install-pip${pv} $*"
+    local name="$*"
+    name="${name// /^}"
```

Comment:
name has been substituted here and below??

Reply:
Forgot that